test/UnbrandedProjects/Unbranded-TypeSpec/src/Generated/UnbrandedTypeSpecClient.cs

```diff
@@ -1257,6 +1258,86 @@ public virtual ClientResult<bool> HeadAsBoolean(string id, RequestOptions option
             return ClientResult.FromValue(result.Value, result.GetRawResponse());
         }
 
+        /// <summary> put array. </summary>
+        /// <param name="body"> The <see cref="IEnumerable{T}"/> where <c>T</c> is of type <see cref="string"/> to use. </param>
+        /// <exception cref="ArgumentNullException"> <paramref name="body"/> is null. </exception>
+        public virtual async Task<ClientResult> PutArrayAsync(IEnumerable<string> body)
```

Comment:
If we own this test case, it might be helpful to return an output model to make it clear this is a convenience method.  Sometimes PUT returns the updated resource in the response body, so this wouldn't be entirely unnatural.

Reply:
This is only added to show that we still need these two files in non-azure libraries.
I will add some more context in the other comment

Reply:
Yes we have a lot similar cases in cadl-ranch, I also introduced them in this PR for non-azure cases